### PR TITLE
feat: add support for disable tcp reset

### PR DESCRIPTION
**Pull request by feiskyer**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:

Add Service annotation `service.beta.kubernetes.io/azure-load-balancer-disable-tcp-reset` to support disable tcp reset.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
LoadBalancer tcp reset could be disabled with Service annotation "service.beta.kubernetes.io/azure-load-balancer-disable-tcp-reset=true"
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/consts/consts.go [hunk 1]
```diff
@@ -333,6 +333,9 @@ const (
 	// The list is separated by comma. It will be omitted if multi-slb is not used.
 	ServiceAnnotationLoadBalancerConfigurations = "service.beta.kubernetes.io/azure-load-balancer-configurations"
 
+	// ServiceAnnotationDisableTCPReset is the annotation used on the service to disable TCP reset on the load balancer.
+	ServiceAnnotationDisableTCPReset = "service.beta.kubernetes.io/azure-load-balancer-disable-tcp-reset"
```
Comment (MartinForReal) [inline pkg/consts/consts.go:337]: I'm wondering if we need to add annotation key for port in line 457

Comment (feiskyer) [reply pkg/consts/consts.go:337]: No need for specific ports. It applies to all the ports on the service.

**Pull request merged**